lib/libs/email/content/email-components.tsx

```diff
@@ -197,27 +205,39 @@ const SpamNotice = () => (
 );
 
 const ContactStateLead = ({ isChip }: { isChip?: boolean }) => (
-  <Section
-    style={{
-      ...styles.section.footer,
-      paddingLeft: "16px",
-      paddingRight: "16px",
-    }}
-  >
+  <Section>
+    <Divider />
     <Text style={{ fontSize: "14px" }}>
       If you have questions or did not expect this email, please contact{" "}
       <Link
         href={`mailto:${isChip ? EMAIL_CONFIG.CHIP_EMAIL : EMAIL_CONFIG.SPA_EMAIL}`}
-        style={{ color: "#fff", textDecoration: "underline" }}
+        style={{ textDecoration: "underline" }}
       >
         {isChip ? EMAIL_CONFIG.CHIP_EMAIL : EMAIL_CONFIG.SPA_EMAIL}
       </Link>{" "}
       or your state lead.
     </Text>
-    <Text>Thank you!</Text>
+    <Text>Thank you.</Text>
   </Section>
 );
 
+export const SpamWarning = () => {
+  return (
+    <Section>
+      <Divider />
+      <Text style={{ fontSize: "14px" }}>
+        If the contents of this email seem suspicious, do not open them, and instead forward this
+        email to{" "}
+        <Link style={{ textDecoration: "underline" }} href="mailto:SPAM@cms.hhs.gov">
```

Comment:
```suggestion
        <Link style={{ textDecoration: "underline" }} href={`mailto:${EMAIL_CONFIG.SPAM_EMAIL}`}>
```

We should use the source of truth for the spam warning